app/src/main/res/values/styles.xml

```diff
@@ -63,6 +63,19 @@
     <item name="fontFamily">@font/default_typeface_regular</item>
   </style>
 
+  <style name="Schedule.EmptyView.TextAppearance" parent="None">
```

Comment:
This should go after its usage, not before — and the name should be `TextAppearance.Squanchy.Schedule.EmptyView` for consistency with the project conventions for TextAppearance

Reply:
796e1fb